src/ngx_http_lua_timer.c

```diff
@@ -54,9 +58,92 @@ ngx_http_lua_inject_timer_api(lua_State *L)
     lua_pushcfunction(L, ngx_http_lua_ngx_timer_at);
     lua_setfield(L, -2, "at");
 
```

Comment:
We need to adjust the "nrec" size for the lua_createtable statement earlier in this function.